ui-tests/src/test/java/org/openmrs/reference/page/PatientCaptureVitalsPage.java

```diff
@@ -138,6 +139,13 @@ public void confirm() {
             confirmButton.click();
         } catch (Exception e) {}
     }
+    
+    public void saveChanges() {
+        try {
+            WebElement saveChangesButton = findElement(SAVE_CHANGES_BUTTON);
+            saveChangesButton.click();
+        } catch (Exception e) {}
```

Comment:
```Some thing this 
        } catch (Exception e) {
        e.getMessage("failed to save the changes")
        }

Its not a good practice to leave the empty blackets in catch block 
```

Reply:
@sherrif10 could you respond to [this please](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/562#discussion_r709809056) so that I can make another commit

Reply:
@sherrif10 this  e.getMessage("failed to save the changes") can not work with this direct string. Are there predefined return messages I could just call?

Reply:
For now, feel free to leave it the way your code is, then we shall handle that return types in a separate pull request